src/OSPSuite.Core/Domain/Constants.cs

```diff
@@ -207,7 +207,7 @@ public static class Filter
          public static readonly string CHART_LAYOUT_FILTER = XmlFilter("Chart Layout");
          public static readonly string PDF_FILE_FILTER = FileFilter("Report", PDF_EXTENSION);
          public static readonly string XML_FILE_FILTER = XmlFilter("Xml");
-         public static readonly string EXCEL_SAVE_FILE_FILTER = string.Format("Excel file (*{0})|*{0}|(Excel file  *{1})|*{1}", XLS_EXTENSION, XLSX_EXTENSION);
+         public static readonly string EXCEL_SAVE_FILE_FILTER = string.Format("Excel file (*{0})|*{0}|(Excel file  *{1})|*{1}", XLSX_EXTENSION, XLS_EXTENSION);
```

Comment:
Please also move the bracket: 
![clipboard01](https://cloud.githubusercontent.com/assets/25061876/25182338/8f1c84d0-2514-11e7-95cd-f4b8d827c00c.jpg)